### DISCUSSION TITLE
k8s version 1.24 service account token generation

### DISCUSSION THREAD

**Discussion by prakashpsgcse** [strimzi/strimzi-kafka-operator#10214] [General]
In k8s version 1.24 service account tokens are not auto generated . Is it handled in Strimzi operator ?  

FYI: We are trying to upgrade  version 0.33.2 from 0.29.0 

https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/2799-reduction-of-secret-based-service-account-token#summary

**Comment by scholzj** [comment]
I don't think there is any need to handle that. This is as far as I know about Service Account Secrets. That said, you are talking about very old versions, so the best think you can do is test it your self in your environment.

**Reply by prakashpsgcse** [reply]
We have tested and you are right about service account secrets. I thought in Fabrick8  client uses  secret from service account directly. Looks like it follow same as other clients ,uses token in path  /var/run/secrets/kubernetes.io/serviceaccount/token generated  by k8s for service acc which pod is configured. 

Thanks you so much for clarifying.
